pkg/build/controller/controller_test.go

```diff
@@ -390,14 +399,11 @@ func TestHandleBuild(t *testing.T) {
 		if tc.inStatus != buildapi.BuildPhaseError && build.Status.Phase == buildapi.BuildPhaseError && len(build.Status.Message) == 0 {
 			t.Errorf("(%d) errored build should set message: %#v", i, build)
 		}
-		if len(tc.outputSpec) != 0 {
-			build := ctrl.BuildStrategy.(*okStrategy).build
```

Comment:
Did you figure out why this line was here? Keep in mind that removing it can change the meaning of the tests.